CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-## Contributing
+## Contributing to the _GitHub Extension for Visual Studio_
```

Comment:
Actually, nevermind! My brain was telling me this was in the issue template, but it's actually in the contributing document, which is rendered. bleh.